react/week3/src/initialData.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useCallback } from "react";
+
+const DataFetching = (props) => {
```

Comment:
Since this is only used for fetching your data, it could just be a function that will be called from a useEffect hook inside your App.js and returns you your initial data, so you can set it there.
Otherwise, seems a bit unnecessary to  return <> </>